lib/pure/mimetypes.nim

```diff
@@ -14,481 +14,1862 @@ type
-    "bcpio": "application/x-bcpio",
-    "torrent": "application/x-bittorrent",
-    "cab": "application/x-cab",
-    "cbr": "application/x-cbr",
-    "cbz": "application/x-cbz",
-    "cdf": "application/x-cdf",
-    "cda": "application/x-cdf",
-    "vcd": "application/x-cdlink",
-    "pgn": "application/x-chess-pgn",
-    "cpio": "application/x-cpio",
-    "csh": "application/x-csh",
-    "deb": "application/x-debian-package",
-    "udeb": "application/x-debian-package",
-    "dcr": "application/x-director",
-    "dir": "application/x-director",
-    "dxr": "application/x-director",
-    "dms": "application/x-dms",
-    "wad": "application/x-doom",
-    "dvi": "application/x-dvi",
-    "rhtml": "application/x-httpd-eruby",
-    "pfa": "application/x-font",
-    "pfb": "application/x-font",
-    "gsf": "application/x-font",
-    "pcf": "application/x-font",
-    "pcf.Z": "application/x-font",
-    "mm": "application/x-freemind",
-    "spl": "application/x-futuresplash",
-    "gnumeric": "application/x-gnumeric",
-    "sgf": "application/x-go-sgf",
-    "gcf": "application/x-graphing-calculator",
-    "gtar": "application/x-gtar",
-    "tgz": "application/x-gtar",
-    "taz": "application/x-gtar",
-    "hdf": "application/x-hdf",
-    "phtml": "application/x-httpd-php",
-    "pht": "application/x-httpd-php",
-    "php": "application/x-httpd-php",
-    "phps": "application/x-httpd-php-source",
-    "php3": "application/x-httpd-php3",
-    "php3p": "application/x-httpd-php3-preprocessed",
-    "php4": "application/x-httpd-php4",
-    "php5": "application/x-httpd-php5",
-    "ica": "application/x-ica",
-    "info": "application/x-info",
-    "ins": "application/x-internet-signup",
-    "isp": "application/x-internet-signup",
-    "iii": "application/x-iphone",
-    "iso": "application/x-iso9660-image",
-    "jam": "application/x-jam",
-    "jnlp": "application/x-java-jnlp-file",
-    "jmz": "application/x-jmol",
-    "chrt": "application/x-kchart",
-    "kil": "application/x-killustrator",
-    "skp": "application/x-koan",
-    "skd": "application/x-koan",
-    "skt": "application/x-koan",
-    "skm": "application/x-koan",
-    "kpr": "application/x-kpresenter",
-    "kpt": "application/x-kpresenter",
-    "ksp": "application/x-kspread",
-    "kwd": "application/x-kword",
-    "kwt": "application/x-kword",
-    "latex": "application/x-latex",
-    "lha": "application/x-lha",
-    "lyx": "application/x-lyx",
-    "lzh": "application/x-lzh",
-    "lzx": "application/x-lzx",
-    "frm": "application/x-maker",
-    "maker": "application/x-maker",
-    "frame": "application/x-maker",
-    "fm": "application/x-maker",
-    "fb": "application/x-maker",
-    "book": "application/x-maker",
-    "fbdoc": "application/x-maker",
-    "mif": "application/x-mif",
-    "wmd": "application/x-ms-wmd",
-    "wmz": "application/x-ms-wmz",
-    "com": "application/x-msdos-program",
-    "exe": "application/x-msdos-program",
-    "bat": "application/x-msdos-program",
```

Comment:
Now it's "text/plain"?

Reply:
Done :+1:

Reply:
Well yes, but now I'm skeptical about your approach. Can't the be generated from some official spec?

Reply:
They are from the linked source, nothing shady on the code.
I generate it from official spec, they are not made up.

But the official-ity of the official is also kinda relative, along history extensions have been reused, etc.
The same happens across other langs too, Ive been checking Rust, NodeJS, Python, Go, and their Mimes are not exactly the same, they are a lot different.

The point is to keep the well known extensions mapped to its mimes.

Reply:
Ok.